mmdet/apis/test.py

```diff
@@ -64,6 +64,13 @@ def single_gpu_test(model,
         if isinstance(result[0], tuple):
             result = [(bbox_results, encode_mask_results(mask_results))
                       for bbox_results, mask_results in result]
+        # This logic is only used in panoptic segmentation test.
+        elif isinstance(result[0], dict) and 'ins_results' in result[0]:
```

Comment:
Does this only used in `panoptic segmentation test.` or only used in `mask2former`? 
Is it used in Panoptic FPN?

Reply:
This logic is used in `panoptic segmentation test`. Models which return panoptic segmentation results and instance segmentation results simultaneously will use this.